lms/models/json_settings.py

```diff
@@ -85,6 +84,9 @@ def get(self, group: str, key: str, default=None):  # type: ignore[override]
         """
         return super().get(group, {}).get(key, default)
 
+    def get_setting(self, setting: JSONSetting):
```

Comment:
This get the value of a setting based on JSONSetting, it used the default on that object.